independent-publisher-connectors/Notion/apiDefinition.swagger.json

```diff
@@ -0,0 +1,677 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Notion",
+    "description": "Notion connector use the API to access Notion's pages, databases, and users.",
+    "version": "1.0",
+    "contact": {
+      "name":"Chandra Sekhar & Harshini Varma",
+      "url":"https://github.com/sekharmalla",
+      "email":"Sekharmalla.vfx@gmail.com"
+    }
+  },
+  "host": "api.notion.com",
+  "basePath": "/v1/",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/databases/c9bc0f8b735f4b089c91e8a5e93df396/query": {},
+    "/databases/493fb17afa214f1b8b46c243894350c9/query": {},
+    "/databases/items": {},
+    "/databases/493fb17a-fa21-4f1b-8b46-c243894350c9/query/item": {},
+    "/databases/": {},
+    "/databases/database_id": {},
+    "/databases": {},
```

Comment:
@sekharmalla You have not removed all the empty operations. Please push all the changes and have one round of check of your connector artifacts before replying back to the comment or PR.

Reply:
Sorry for the delay. i have made the changes. As you said i have pushed all changes once and marking now as resolved. Thanks for the tip.